cep-0012.md

```diff
@@ -52,11 +52,11 @@ The schema of `run_exports.json` will mimic the `repodata.json` structure whenev
 
 Each `run_exports` metadata `dict` can contain the following fields; each field accepts a list of strings (conda-build specs).
 
-- `weak`
-- `strong`
-- `weak_constrains`
-- `strong_constrains`
-- `noarch`
+* `weak`
```

Comment:
Do we know why `*` or `-` is preferred? Seems we are switching some to one or the other, but without a clear reason

Reply:
See my comment above. Use the same symbol everywhere and you will be fine.